modules/tinymce/src/models/dom/main/ts/table/actions/Clipboard.ts

```diff
@@ -45,13 +44,16 @@ const registerEvents = (editor: Editor, selections: Selections, actions: TableAc
     };
 
     if (e.selection === true) {
-      SelectionTypes.cata(selections.get(), Fun.noop, multiCellContext, Fun.noop);
+      const cells = TableSelection.getCellsFromSelection(editor);
+      if (cells.length >= 1) {
```

Comment:
This doesn't look correct... the old version only ran this when `cells.length > 1` and did a noop on a single cell selection. It's possible `getCellsFromSelection` has changed, so can you just double check this is correct please?

Reply:
Hmm, you might be right. Making a ranged text selection within a cell and copying isn't working the same as fiddle. I'll have  a look into this more and see if I can figure out and beter solution

Reply:
I have added put together a solution for this to make it operate the same as the old code